x-pack/legacy/plugins/index_management/public/app/services/documentation.ts

```diff
@@ -56,9 +55,25 @@ class DocumentationService {
     return `${this.esDocsBase}/mapping-types.html`;
   }
 
+  public getDynamicMappingLink() {
```

Comment:
I am not quite sure about all those methods but it deserves that we talk about it over zoom to explain my reasoning. Let's leave them for now 😊